content/departments/product-engineering/engineering/policies/vulnerability-management-policy.md

```diff
@@ -0,0 +1,142 @@
+# Vulnerability Management Policy
+
+## Introduction
+
+This policy defines Sourcegraph's process for incoming security vulnerabilities. It defines roles, responsibilities, steps and SLAs.
+
+This document is [approved by the CEO and VP of Engineering](https://github.com/sourcegraph/handbook/pull/1352). The document is managed through git and history can be found in the git repository.
+
+## Scope
+
+This document concerns how we handle incoming vulnerabilities and engage with Engineering teams to ensure they are fixed. It does not cover tooling and processes to find vulnerabilities. More information on how we execute this process can be found on the Vulnerability Management Process document. The policy document is the source of truth in case there are any discrepancies between the two documents. If you find conflicting information please raise it to the attention of the Security team.
```

Comment:
I assume we will add a link to "Vulnerability Management Process document" once it is checked in? Can we add a link to the Google Doc in the meantime?

Reply:
Yup I'm moving the process doc to the handbook and will fix the link here once that's done. In the meantime I linked the Google doc.